File: 2390. Removing Stars From a String/solution.cpp
class Solution {
public:
    string removeStars(string s){
        stack<char> c;
        for(auto i: s){
            c.push(i);
            if(i == '*'){
                c.pop();
                c.pop();
            }
        }
        string s2;
        while(!c.empty()){
            s2 += c.top();
            c.pop(); 
        }
        reverse(s2.begin(), s2.end());

        return s2;
    }
}

File: 2390. Removing Stars From a String/solution.py
class Solution:
    def removeStars(self, s: str) -> str:
        stack = []

        for i in s:
            stack.append(i)
            if i == '*':
                stack.pop()
                stack.pop()
        
        s2 = ""
        while stack:
            s2 += stack.pop()

        return s2[::-1]
        
        